packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -72,6 +77,8 @@ export class IosSimulatorDevice extends DeviceBase {
 
   public dispose() {
     super.dispose();
+    this.nativeLogsOutputChannel?.dispose();
+    this.lunchedAppCancelToken.cancel();
```

Comment:
cancel token can only be cancelled once. We should create new one for each process launch